middlewares/error_pages_test.go

```diff
@@ -0,0 +1,66 @@
+package middlewares
```

Comment:
It might be nice if we could also have an integration test for this, similar to [the healthcheck one](https://github.com/bparli/traefik/blob/64e8b31d4956e26886463a24094148542afd9717/integration/healthcheck_test.go).